tests/test_well/conftest.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from xtgeo import Well
+
+
+@pytest.fixture()
+def string_to_well(setup_tmpdir):
```

Comment:
Generally, this seems technically the way to go, ie. dump it to file first before loading it in the test, but I think the test would be more readable if it went through an object that carried the structured data first:
```python
@dataclass
class RMSWell:
   name: str
   ypos: float
   xpos: float
   rkb: Optional[float]
   num_logs: int
   well_logs: List[WellLog]
   version: str = "1.01"
   well_type: str = "Unknown"
   
   def to_file(self, filename):
      ...   
      
class WellLog:
   name: str
   log: pd.DataFrame

```

Reply:
As suggested, we should do this in a separate PR.